bgpd/bgp_nexthop.c

```diff
@@ -47,6 +47,11 @@
 
 DEFINE_MTYPE_STATIC(BGPD, MARTIAN_STRING, "BGP Martian Address Intf String");
 
+static void bgp_ipv6_address_add(struct bgp *bgp, struct connected *ifc,
+					struct prefix *p);
```

Comment:
This commit has a variety of formatting issues, including a weirdly named file!  Please take the time and address all of them.  I'm not going to waste my time continuing looking at code we can't even accept in it's current format.

Reply:
@donaldsharp  There was some problem with my local branch, so the bgp_nexthop.c file got corrupted during push operation. I had removed the bgp_nexthop.c, rebased and added the changes to this file again. Now everything should be fine